helm/ovn-kubernetes/values.yaml

```diff
@@ -9,6 +9,9 @@ tags:
   ovnkube-node-dpu-host: false
   ovnkube-single-node-zone: false
   ovnkube-zone-controller: false
+  ovnkube-db: true
```

Comment:
Are these 3 lines needed? iirc, only the instance we are setting to `false` needed to be mentioned here.

Reply:
I verified that these 3 lines are not needed, removed.